libcxx/cmake/caches/NVPTX.cmake

```diff
@@ -0,0 +1,36 @@
+# Configuration options for libcxx.
+set(LIBCXX_ABI_VERSION 2 CACHE STRING "")
```

Comment:
Can you sort the entries? Same in the other file.

Reply:
Sure, I'm guessing sorted alphabetically?

Reply:
Yes